### DISCUSSION TITLE
Lanes for Multi-Lane roundabout

### DISCUSSION THREAD

**Discussion by HafMann** [organicmaps/organicmaps#4706] [Ideas]
Hi, I would like for organic maps to update roundabout signs to show the correct lane placement whether it is the left or right one similar to how Google Maps does it.

**Comment by biodranik** [comment]
Aren't there already some existing issues?

**Reply by Altonss** [reply]
@HafMann Are you looking for something like this https://github.com/organicmaps/organicmaps/issues/2281 ?
